src/burn/engine/core.cpp

```diff
@@ -508,17 +508,23 @@ extern "C" HRESULT CoreElevate(
     )
 {
     HRESULT hr = S_OK;
+    DWORD cAVRetryAttempts = 0;
 
-    // If the elevated companion pipe isn't created yet, let's make that happen.
-    if (INVALID_HANDLE_VALUE == pEngineState->companionConnection.hPipe)
+    while (INVALID_HANDLE_VALUE == pEngineState->companionConnection.hPipe)
     {
+        // If the elevated companion pipe isn't created yet, let's make that happen.
         if (!pEngineState->sczBundleEngineWorkingPath)
         {
             hr = CacheBundleToWorkingDirectory(pEngineState->registration.sczId, pEngineState->registration.sczExecutableName, &pEngineState->userExperience.payloads, &pEngineState->section, &pEngineState->sczBundleEngineWorkingPath);
             ExitOnFailure(hr, "Failed to cache engine to working directory.");
         }
 
         hr = ElevationElevate(pEngineState, hwndParent);
+        if (E_SUSPECTED_AV_INTERFERENCE == hr && 1 > cAVRetryAttempts)
+        {
+            ++cAVRetryAttempts;
```

Comment:
We increment this var, but do nothing else with it? Why have it at all?

Reply:
I think you're missing it in the `if` statement two lines above? Without the `continue` right below this, it would break out of the loop and not try again.

Reply:
The if statement isn't using cAVRetryAttempts, and it doesn't appear to reference anything that does.